### DISCUSSION TITLE
is AlDente right for my use?

### DISCUSSION THREAD

**Discussion by jutah76** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#976] [Q&A] [answered]
Hi, i just upgrade my old MBP to a new macbook pro M2.
I use it almost with the lid closed, connected to a docking station that charge the laptop with a tb3 cable,.
The battery is obviously always at 100%.

I've read the faq but i don't understand this: when i finish my working week, friday, my battery will be at 80%. What will happen during the following 48hours, till monday?

The battery will discharge and AlDente can't be able to keep the mbp at 80&, am i wrong?

Which kind of settings do you suggest me in order to save my battery over time?
I'll keep this machine for other 4 years also.

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @jutah76,

If you are using the MacBook mostly plugged in, it is a good idea to use AlDente for setting a charge limit to reduce battery aging.
Are you using AlDente Free or did you buy AlDente Pro?

If you are using AlDente Free, when your MacBook stays plugged in over the weekend and charging was paused at the charge limit and you do not turn off your MacBook, it will stay at 80%. However, if you turn off your MacBook, it will start charging to 100%.
If you bought AlDente Pro, you have more features available, and charging does not need to be paused when you leave your MacBook on Fridays. When you have features such as "Stop Charging when Sleeping" or "Disable Sleep until Charge Limit" enabled, your MacBook will not charge to 100% when you leave your MacBook while it is charging. However, even with AlDente Pro, when you turn off your MacBook, it will charge to 100% again.

You can read more about these features on our blog [here](https://apphousekitchen.com/category/featureexplanation/).

Best,
Matthias
